src/OSPSuite.Core/Services/PredictedVsObservedChartService.cs

```diff
@@ -120,6 +120,11 @@ public void ConfigureAxesDimensionAndTitle(IReadOnlyList<DataColumn> observation
          deviationCurves.Where(repository => repository != null).Each(repository =>
             chart.AddCurvesFor(repository, x => x.Name, _dimensionFactory, (column, curve) => curve.UpdateDeviationCurve(column.Name, deviationLinesCount)));
 
+         //adding one of the deviation lines to the legend. Workaround for now until the presenter is adjusted for Predicted vs. Observed Chart.
+         var upperDeviationLine = chart.Curves.FirstOrDefault(curve => curve.Name.Equals(Captions.ParameterIdentification.DeviationLineNameUpper(foldValue)));
```

Comment:
use FindByName here
Also please import statics  Captions.ParameterIdentification
Lastly, this DeviationLineUpper function should not be under the ParameterIdentfication "class" because it's user outside of it. If you are on it already?

Reply:
My personal taste is to have the captions.... namespace by the call - we also have it like that all over the place, but if you insist I can change it.

Everything else is fixed